src/serac/numerics/solver_config.hpp

```diff
@@ -282,7 +300,15 @@ struct NonlinearSolverOptions {
   /**
    * @brief Nonlinear solver selection
    */
-  NonlinearSolver nonlin_solver = NonlinearSolver::MFEMNewton;
+  NonlinearSolver nonlin_solver = NonlinearSolver::Newton;
 };
 
+/**
+ * @brief Parameters for a nonlinear solver
+ *
+ * This can either be a user-constructed and owned solver or an iterative solver that serac constructs and builds
+ * itself.
+ */
+using NonlinearSolverOptions = std::variant<CustomNonlinearSolverOptions, IterativeNonlinearSolverOptions>;
```

Comment:
Can you explain why one is "Custom" and the other is not?

Reply:
Custom means a user builds their own MFEM solver object and passes it in as a raw pointer. I can add some better verbiage on this.